### DISCUSSION TITLE
Relay status text changes if more than 4 defined

### DISCUSSION THREAD

**Discussion by Pewibe** [arendst/Tasmota#11619] [Q&A] [answered]
I run a Sonoff DualR2 with 4 relays in sum (see 1st screen).
![Clipboard04](https://user-images.githubusercontent.com/42893315/113877392-f4b1c600-97b8-11eb-9a8b-917955c80946.jpg)
Extending the config with a 5th relay I discovered that the text showing the relay status changed from ON/OFF to 0/1 (see 2nd screen).
![Clipboard05](https://user-images.githubusercontent.com/42893315/113877407-f8454d00-97b8-11eb-82b7-5fc3f7081447.jpg)
Is this a documented feature which I didn't read? Can it be documented (with the idea behind) if undocumented feature? If a bug which will be solved in Tasmota: Can it be made configurable like "WebButtonx <text>"?

**Comment by sfromis** [comment]
I'd assume that it is to allow for a compact display when up to 28 relays (and a few other outputs) are represented at the same time, where the default text changes to just the number of the output, instead of "Toggle 1" and so on. Of course, the compactness breaks down when you yourself assign a longer text to replace the default one.

**Comment by arendst** [comment, marked as answer]
Indeed as designed for compactness on mobile phone.

**Comment by Pewibe** [comment]
OK, so away from guessing and back to my question: This is an undocumented feature! If I like to update the doc, do I have to know more than @sfromis / @arendst have written?
Additional question: Why not leaving these very seldom applications (e.g. more than 4 relays) up to the user instead of writing code which provides some kind of automatic exception? If I was asking for any small feature for Tasmota in the past, the first argument against it was the growing of code. Yes, I like philosophical discussions.
